<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REGISTER</class>
 <widget class="QWidget" name="REGISTER">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CREATE USER:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CREATE PASSWORD:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_user">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_passw">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CONFIRM PASSWORD:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnf_passw">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BOTON_REGI2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
